### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by agascon9**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -99,4 +99,45 @@ deploy-ltqa-v3: deploy-v3
...
+	    --stack-name account-emails-alarms-$(BANNER) \
+	    --capabilities CAPABILITY_NAMED_IAM \
+	    --template-body file://cloudformation/alarms/$(BANNER)-template.json \
+	    --notification-arns $(NOTIFICATION_ARN) \
+	    --tags Key=Team,Value=TRex
+
+create-alert-stack-saks: BANNER = saks
+create-alert-stack-saks: create-alerts-stack
+
+create-alert-stack-lt: BANNER = lt
```
Comment (tpon) [inline Makefile]: dont think we need this for LT as its not going to salesforce

File: Makefile [hunk 2]
```diff
@@ -99,4 +99,45 @@ deploy-ltqa-v3: deploy-v3
...
+create-alert-stack-off5th: BANNER = off5th
+create-alert-stack-off5th: create-alerts-stack
+
+create-alert-stack-bay: BANNER = bay
+create-alert-stack-bay: create-alerts-stack
+
+update-alert-stack-saks: BANNER = saks
+update-alert-stack-saks: update-alerts-stack
+
+update-alert-stack-lt: BANNER = lt
```
Comment (tpon) [inline Makefile]: dont think we need LT alerts here either

File: cloudformation/alarms/lt-template.json [hunk 3]
```diff
@@ -0,0 +1,350 @@
+{
```
Comment (tpon) [inline cloudformation/alarms/lt-template.json]: don't think we'll need this as LT won't be going to salesforce

File: src/main/scala/com.hbc.email/ProcessPayload.scala [hunk 4]
```diff
@@ -32,13 +35,22 @@ trait ProcessPayload extends AccountEmailJsonParser with AccountEmailDatabaseUpd
...
       payload
+    case (_, _) =>
+      Logger[IO].info(s"Failed To Parse Account Email: $payload").unsafeRunSync()
+      payload
+  }
+
+  private[email] def writeCloudwatchMetrics(response: Validate[Response], accountEmail: Validate[AccountEmail], metrics: Metrics): Validate[Unit] = (response, accountEmail) match {
+    case (Good(_: SuccessResponse), Good(ae)) => Good(metrics.incrementEmailSentCount(ae.banner, ae.event_subtype))
+    case (_, Good(ae)) => Good(metrics.incrementEmailErrorCount(ae.banner, ae.event_subtype))
+    case _ => Bad(One(FailedToWriteCloudwatchMetrics()))
```
[the remark this answers is not in the document]
Comment (agascon9) [reply src/main/scala/com.hbc.email/ProcessPayload.scala]: I didn't create translation error metrics - I'll add those.  Line 52 is incrementing the error count when an email request fails to be generated/fails to be sent to Cheetah - but this supposes a successfully parsed account email.

**Pull request merged**